cypress/support/customCypressUtils.ts

```diff
@@ -7,32 +7,27 @@ declare global {
        * Custom command to select DOM element by data-cy attribute.
        * @example cy.dataCy('greeting')
        */
-      dataCy(options: {
-        value: string;
-        timeout?: number;
-      }): Chainable<JQuery<HTMLElement>>;
-      dataId(options: {
-        value: string;
-        timeout?: number;
-      }): Chainable<JQuery<HTMLElement>>;
+      dataCy(value: string, timeout?: number): Chainable<JQuery<HTMLElement>>;
```

Comment:
another case: 
`options: {` <--- removed and reformatted
        value: string;
        timeout?: number;
})

Reply:
Yeah, I did this manually, as specified in the last comment